src/thunderbird_accounts/authentication/tests.py

```diff
@@ -1,32 +1,77 @@
+import uuid
+
 from django.conf import settings
+from django.core.cache import cache
 from django.http import HttpResponseRedirect
-from django.test import TestCase
+from django.test import TestCase, RequestFactory
 from rest_framework.test import RequestsClient
 
+from django.contrib.auth import get_user_model
+
 from thunderbird_accounts.authentication.models import User
 from thunderbird_accounts.authentication.utils import handle_auth_callback_response
 from thunderbird_accounts.client.models import Client, ClientEnvironment
 
+from thunderbird_accounts.authentication.middleware import FXABackend, ClientSetAllowedHostsMiddleware
+
 
 class APITestCase(TestCase):
     def setUp(self):
+        self.GET_LOGIN_PATH = 'http://testserver/api/v1/auth/get-login/'
+        self.BAD_CREDENTIALS_MSG = 'Authentication credentials were not provided.'
```

Comment:
nit: Maybe put these in a separate `const.py` file, then the might be easier to find if they change? Although there's only the two values so no biggie!

Reply:
I think I might hold off for the moment? I wonder if these values are specific enough to the test that they could stay for now.

Reply:
I'm fine with these being test class level...at least until we use them in more than one test class 🤔